webrtc/videoroom/assets/src/membraneWebRTC.ts

```diff
@@ -101,12 +101,31 @@ export class MembraneWebRTC {
     this.connection = undefined;
   };
 
+  public muteMicrophone = async () => {
+    this.localTracks.forEach(([track, stream]) => {
+      if (track.kind == "audio") {
+        track.enabled = !track.enabled;
+        this.channel.push("mute-microphone", { data: !track.enabled, mid: this.getSendTrackMid(track) })
+      }
+    });
+    var buttonText = document.getElementById("mute-microphone")?.innerText
```

Comment:
Don't use `var`. Use `let` if you intend to change this variable reference and `const` otherwise.